app/services/foreman_templates/template_importer.rb

```diff
@@ -133,21 +133,17 @@ def update_job_template(name, text)
     end
 
     def purge!
-      # Available options:
-      #* verbose => Print extra information during the run [false]
-      #* not     => negate prefix / filter
-      #* kind    => Template kind
-      #* prefix  => The string all imported templates should begin with [Community ]
-      #* filter  => Import names matching this regex (case-insensitive; snippets are not filtered)
-
-      @verbose = ( ENV['verbose'] and ENV['verbose'] != 'false' ) ? true : false
-      match    = ENV['not'] ? 'NOT LIKE' : 'LIKE'
-      #kind     = ENV['kind'] ? ENV['kind'] : "template_kind #{match} '%'"
-      prefix   = ENV['prefix'] ? ENV['prefix'] : 'Community '
-      filter   = ENV['filter'] ? ENV['filter'] : "name #{match} '#{prefix}%'"
+#      @kind    = args[:kind] || 'something'
+      @match   = args[:negate] ? 'NOT LIKE ' : 'LIKE '
```

Comment:
I'd still leave this out if you're not going to use it later on - it's just a code-clarity thing.